src/main/resources/cgmes_assembling.cql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS cgmes_assembling.handled_files (
+    filename text,
+    origin text,
+    import_date timestamp,
```

Comment:
The values stored in this import_date column are currently not read in code, but I guess it might be useful in the future. But the column name might be changed to handling_date or something like that ?

Reply:
Done